packages/lesswrong/components/tagging/WikiTagNestedList.tsx

```diff
@@ -48,7 +48,8 @@ const WikiTagNestedList = ({
         <ConceptItem 
           key={page._id} 
           wikitag={page} 
-          nestingLevel={nestingLevel}
+          // TODO: this will be broken but who knows if we're even keeping this component
```

Comment:
?

Reply:
WikiTagNestedList isn't used in the current state of things. Was used in previously fully nestable Arbital Explore page

Reply:
I thought about removing the whole component but there's a bit of a cascade conceivably we want something from it